templates/content-fr.md

```diff
@@ -6,6 +6,8 @@ altLangPage: content-en.html
 secondlevel: false
 dateModified: 2017-09-29
 share: true
+feedbackPath: "/sites/feedback/ajax/report-problem-fr.html"
```

Comment:
Use a relative path instead. It won't work if the site run in a sub directory like GCWeb on github.
```suggestion
feedbackPath: "../sites/feedback/ajax/report-problem-fr.html"
```